### PR TITLE
Add dammit wrapper

### PR DESCRIPTION
**Pull request by bluegenes**
two wrappers: one for databases, one for annotate (as requested on original bitbucket PR)

### CODE REVIEW

**Comment by johanneskoester** [comment]
Hey @bluegenes great to see you back in action! I am looking into the unrelated CI issues now. Once they are solved, I will get back to you.

**Comment by johanneskoester** [comment]
@bluegenes please update to latest master for all your PRs.

File: bio/dammit/annotate/environment.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - shmlast 
+  - infernal 
+  - hmmer 
+  - transdecoder=3.0.1 
+  - last
+  - busco=3.0.2 
+  - parallel 
+  - bioconductor-seqlogo
+  - pip:
+    - git+https://github.com/dib-lab/dammit
```
Comment (johanneskoester) [inline bio/dammit/annotate/environment.yaml]: Can you provide this package via bioconda please?

Comment (bluegenes) [reply bio/dammit/annotate/environment.yaml]: Hi @johanneskoester. We can't seem to fix the bioconda dammit installation https://github.com/bioconda/bioconda-recipes/pull/16305#issuecomment-527583737 - do you have any suggestions? If not, I'll roll back the version, but the previous bioconda install is missing some important functionality.

**Comment by johanneskoester** [comment]
@bluegenes may I ask you to run `black` on your wrappers? Seems like the formatting is not consistent.

**Comment by dlaehnemann** [comment]
@bluegenes It seems like there's no quick and easy way for me to resolve the `test.py` conflict and apply `black` for code formatting. But both should be really quick. Any chance to quickly get this PR merged?

**Comment by johanneskoester** [comment]
Closing because of inactivity. Please reopen if you want to continue on this. Thanks anyway!

**Pull request closed without merging**